homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -35,6 +35,7 @@
 from homeassistant.helpers.entity import Entity, EntityDescription
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.entity_platform import EntityPlatform
+from homeassistant.helpers.frame import report
```

Comment:
Can you use the new `report_usage`?

Reply:
LOL. is there a wild inflation in helpers lately 😆